### ISSUE TITLE
Unet Input

### ISSUE THREAD

**Issue by SshiJwu** [jzr99/Geo4D#10]
I'm a bit confused about how to handle the shape mismatch during initialization. The original video generation model only supports video latent inputs, while Geo4D simultaneously takes four different modality latents as input. When initializing with the pre-trained video generation UNet weights, this causes inconsistency in tensor shapes. Could you explain how you resolved this issue?

**Comment by jzr99** [comment]
Hi, Thank you for your interest.

We modify the first conv layer of the unet model to make it compatible with four different modality latent as input. To be specific, the in_channels are extended from 8 to 20 and the out_channels are extended from 4 to 16 and we use zero initialisation for the additional  in_channel weights.  

Best,
Zeren

**Comment by SshiJwu** [comment]
Thanks for your reply. How the point map are normalized before fed into VAE and what is its coordinate? ​ Is the point-VAE input

a  sequence or single point map?

**Comment by jzr99** [comment]
Hi,

Point map are normalised based on min-max normalisation to make sure the majority of the point map is in the range of [-1, 1], and all the point map is in the reference camera coordinate (first frame). The point-VAE takes a single point map as input.

Best,
Zeren

**Comment by jzr99** [comment]
Hi, 

Thanks again for your interest.

Since this project is still under development, we'll release our training code for both pointmap VAE and diffusion after we finalise the project! Please stay tuned.

Best Wishes,
Zeren

**Comment by SshiJwu** [comment]
Hi, I try to run this project. I use the pretrained point-VAE. But I a bit confused about the uncertainty map. According the loss function, the uncertainty map should be the positive number (because of the log operation). When I run this project, I found that it has negative values.
